### DISCUSSION TITLE
Cannot get to work

### DISCUSSION THREAD

**Discussion by jurr13n** [ExperienceLovelace/ha-floorplan#417] [General]
Hello,
i just installed the floorplan according to the tutorial, and i started to upload the example to the www folder. but i cannot get my floorplan to load in 
<img width="986" alt="image" src="https://github.com/ExperienceLovelace/ha-floorplan/assets/108371655/e949bdb0-8d76-485e-9a8b-909ce69a6e18">

<img width="337" alt="image" src="https://github.com/ExperienceLovelace/ha-floorplan/assets/108371655/84d80b7f-04a8-4a89-9e19-26b6e921b8d3">

i just copy paste the code from the example but still doesnt work

**Comment by OzGav** [comment]
You have loaded the files into /local/home_simple but the error is showing you that the files are being looked for in /local/floorplan/home_simple

**Comment by Trzinka** [comment]
# Hi! If you want, take a look at my https://github.com/ExperienceLovelace/ha-floorplan/discussions/411 how and what. At the end, you also find links to the problems and solution I encountered during the construction of the floor plan card.




![Klikni ta dva gumba](https://github.com/ExperienceLovelace/ha-floorplan/assets/40424965/0079ec25-7713-46fb-9941-56fe5f7ba8df)
